src/Symfony/Component/Workflow/Registry.php

```diff
@@ -50,16 +66,27 @@ public function get($subject, $workflowName = null)
         return $matched;
     }
 
-    private function supports(Workflow $workflow, $className, $subject, $name)
+    /**
+     * @param Workflow $workflow
+     * @param string|SupportStrategyInterface $supportStrategy
+     * @param object $subject
+     * @param null|string $workflowName
+     *
+     * @return bool
+     */
```

Comment:
Several other private methods (e.g. in Workflow.php) have a docblock as well. Is there any rule of thumb when to add it? I personally like having docblocks, but if they are not common in Symfony core I am fine with removing those.